kolibri/plugins/learn/assets/src/state/actions.js

```diff
@@ -288,21 +289,24 @@ function showLearnChannel(store, channelId, page = 1) {
         channelPayload, resumePayload).fetch() : Promise.resolve([]);
       const popularPromise = ContentNodeResource.getCollection(
         channelPayload, popularPayload).fetch();
+      const allContentCollection = ContentNodeResource.getAllContentCollection(
+        channelPayload, { cursor });
+      const allContentPromise = allContentCollection.fetch();
       ConditionalPromise.all(
-        [nextStepsPromise, popularPromise, resumePromise]
+        [nextStepsPromise, popularPromise, resumePromise, allContentPromise]
```

Comment:
Do this synchronously for now - in the future, we may want to load this in dynamically once completed to reduce loading time.